lms/services/blackboard_api/_basic.py

```diff
@@ -80,20 +78,7 @@ def refresh_access_token(self):
         )
 
     def request(self, method, path):
-        url = self._api_url(path)
-
-        try:
-            return self._send(method, url)
-        except ExternalRequestError:
-            if self.refresh_enabled:
-                self._oauth_http_service.refresh_access_token(
-                    self.token_url,
-                    self.redirect_uri,
-                    auth=(self.client_id, self.client_secret),
-                )
-                return self._send(method, url)
-
-            raise
+        return self._send(method, self._api_url(path))
```

Comment:
I don't think `_send` is call anywhere else now. It can probably be inlined now.

Reply:
Done